internal/provider/ses/ses.go

```diff
@@ -39,6 +39,7 @@ func Send(region, username, password, from string, msg *mail.Message) error {
 	}
 
 	input := &ses.SendRawEmailInput{
+		Destinations:         aws.StringSlice(to),
```

Comment:
I think these have to be prefixed with "To: ".

https://docs.aws.amazon.com/sdk-for-go/api/service/ses/#SendRawEmailInput